kubernetes.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: amida-messages-deployment
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: amida-messages
+    spec:
+      imagePullSecrets:
+      - name: regsecret
+      containers:
+      - name: amida-messages
+        image: amidatech/messaging-service
+        env:
+        - name: PG_DB
+          value: amida_messages
+        - name: PG_PORT
+          value: '5432'
+        - name: PG_HOST
+          value: amida-messages.czgzedfwgy7z.us-west-2.rds.amazonaws.com
```

Comment:
We should not commit RDS info to a public repo. Unless this is dummy info, we should cycle/lock down this RDS instance now that the info is public.